### PR TITLE
[BXMSPROD-1945] Set empty next drools and product version

### PR DESCRIPTION
**Pull request by lampajr**
**Thank you for submitting this pull request**

**JIRA**: 

https://issues.redhat.com/browse/BXMSPROD-1945

**referenced Pull Requests**:

- https://github.com/kiegroup/optaplanner/pull/2503
- https://github.com/kiegroup/kie-ci/pull/1408

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

**Review by mareknovotny — APPROVED** [review]

**Review by rgdoliveira — APPROVED** [review]

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 1]
```diff
@@ -42,10 +42,10 @@ def SERVERLESS_LOGIC_DROOLS_CURRENT_PRODUCT_VERSION='8.32.0'
 def SERVERLESS_LOGIC_CURRENT_PRODUCT_BRANCH='1.32.x'
 def SERVERLESS_LOGIC_CURRENT_PRODUCT_CONFIG_BRANCH="openshift-serverless-logic/1.32.x"
 
-def RHBOP_NEXT_PRODUCT_VERSION='8.30.0'
+// def RHBOP_NEXT_PRODUCT_VERSION='8.30.0'
```
Comment (Ginxo) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:45]:
```suggestion
```

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 2]
```diff
@@ -42,10 +42,10 @@ def SERVERLESS_LOGIC_DROOLS_CURRENT_PRODUCT_VERSION='8.32.0'
...
 def RHBOP_NEXT_PRODUCT_BRANCH='main'
 def RHBOP_NEXT_PRODUCT_CONFIG_BRANCH='master'
-def RHBOP_NEXT_DROOLS_VERSION='8.30.0'
+// def RHBOP_NEXT_DROOLS_VERSION='8.30.0'
```
Comment (Ginxo) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:48]:
```suggestion
```

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 3]
```diff
@@ -223,6 +223,24 @@ String rhbopNightlyStage(String version, String droolsVersion, String branch, St
...
+                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: ''],
+                        [\$class: 'StringParameterValue', name: 'DROOLS_PRODUCT_VERSION', value: ''],
+                        [\$class: 'StringParameterValue', name: 'CONFIG_BRANCH', value: "${configBranch}"],
+                        [\$class: 'BooleanParameterValue', name: 'SKIP_TESTS', value: true]
+                ]
+            }
+        }
+    """
+}
+
```
Comment (Ginxo) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:243]:
```suggestion
```

I would avoid this, I would expose instead
```
String rhbopNightlyStage(String branch, String configBranch, String version = '', String droolsVersion = '') {
```

**Review by Ginxo — CHANGES_REQUESTED** [review]
check my suggestions please

**Review by bacciotti — APPROVED** [review]

**Review by Ginxo — APPROVED** [review]
LGTM

**Pull request merged**